releases/pritunl.yaml

```
@@ -59,9 +59,11 @@ releases:
         default:
           internal: 80
           external: 80
-        vpn:
-          internal: {{ env "PRITUNL_VPN_PORT" | default "16807" }}
-          external: {{ env "PRITUNL_VPN_PORT" | default "16807" }}
+      {{- range $i, $port := env "PRITUNL_VPN_PORT" | default "16807" | splitList "," }}
+        vpn{{ $i }}:
+          internal: {{ $port }}
+          external: {{ $port }}
+      {{ end }}
```

Let us by default provision both TCP and UDP ports for the server. It is not required that the user use both, but is, to me, preferable to making the user decide in advance which protocol to use. 

```suggestion
        {{- range $i, $port := env "PRITUNL_VPN_PORT" | default "16807" | splitList "," }}
        vpn-tcp-{{ $i }}:
          internal: {{ $port }}
          external: {{ $port }}
          protocol: TCP
        vpn-udp-{{ $i }}:
          internal: {{ $port }}
          external: {{ $port }}
          protocol: UDP
        {{- end }}
```

---

That is good idea. What do you think about splitting `PRITUNL_VPN_PORT` into 
`PRITUNL_VPN_TCP_PORT` and `PRITUNL_VPN_UDP_PORT` to not exposed ports that won't be used?

---

To provide backward compatibility we could leave `PRITUNL_VPN_PORT` but say it's deprecated.

---

I am in favor of exposing both protocols automatically because they can be switched at the web UI.